src/Sarif/Comparers/ArtifactContentSortingComparer.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+/// <summary>
+/// All Comparer implementations should be replaced by auto-generated codes by JSchema as 
+/// part of EqualityComparer in a planned comprehensive solution.
+/// Tracking by issue: https://github.com/microsoft/jschema/issues/141
+/// </summary>
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal class ArtifactContentSortingComparer : IComparer<ArtifactContent>
+    {
+        internal static readonly ArtifactContentSortingComparer Instance = new ArtifactContentSortingComparer();
```

Comment:
>Instance [](http://example.com/codeflow?start=64&length=8)

not sure, should we also start with s_ since it is static

Reply:
it's a public (internal) member. It should match naming convention: "All visible static member names are pascal cased."